File: android/app/src/main/java/com/interview/tracker/domain/model/Dropdowns.kt
package com.interview.tracker.domain.model

/**
 * Domain model for dropdown options loaded from backend.
 */
data class Dropdowns(
    val joiningAvailability: List<String> = emptyList(),
    val recommendation: List<String> = emptyList(),
    val finalStatus: List<String> = emptyList(),
    val department: List<String> = emptyList(),
    val position: List<String> = emptyList()
)

File: windows/InterviewTracker/InterviewTracker/Models/Dropdowns.cs
using System.Text.Json.Serialization;

namespace InterviewTracker.Models;

public class Dropdowns
{
    [JsonPropertyName("joiningAvailability")]
    public List<string> JoiningAvailability { get; set; } = new();

    [JsonPropertyName("recommendation")]
    public List<string> Recommendation { get; set; } = new();

    [JsonPropertyName("finalStatus")]
    public List<string> FinalStatus { get; set; } = new();

    [JsonPropertyName("department")]
    public List<string> Department { get; set; } = new();

    [JsonPropertyName("position")]
    public List<string> Position { get; set; } = new();
}

